assets/global.js

```diff
@@ -906,6 +906,7 @@ class VariantSelects extends HTMLElement {
 
   renderProductInfo() {
     const requestedVariantId = this.currentVariant.id;
+    const sectionId = this.dataset.originalSection ? this.dataset.originalSection : this.dataset.section;
```

Comment:
I'm not sure what `this.dataset.originalSection` is referring to, but I'm keeping it to be safe. Maybe @LucasLacerdaUX can provide some context since it was added in this [PR](https://github.com/Shopify/dawn/pull/1388/files).